examples/getting_started/main.py

```diff
@@ -1,28 +1,90 @@
+# type: ignore
+
+
 from rabbitmq_amqp_python_client import (
+    AddressHelper,
     BindingSpecification,
     Connection,
+    Event,
     ExchangeSpecification,
     Message,
+    MessageAck,
+    MessagingHandler,
     QuorumQueueSpecification,
-    exchange_address,
 )
 
 
+class MyMessageHandler(MessagingHandler):
+
+    def __init__(self):
+        super().__init__(auto_accept=False)
```

Comment:
I'd create our `MessagingHandler` with `super().__init__(auto_accept=False)` by default. 
All the consumers should use our `MessagingHandler` class